hazelcast/src/main/java/com/hazelcast/jet/impl/util/Util.java

```diff
@@ -647,7 +646,4 @@ public static String formatJobDuration(long durationMs) {
         };
     }
 
-    public static NodeEngineImpl getNodeEngine(JetInstance instance) {
-        return ((HazelcastInstanceImpl) instance.getHazelcastInstance()).node.nodeEngine;
```

Comment:
You could have kept this method, the code in it quite bad-looking and it was used a couple of times.